src/applications/travel-pay/sass/travel-pay.scss

```diff
@@ -25,3 +25,8 @@
   align-items: flex-end;
   column-gap: 20px;
 }
+
+.travel-claim-card {
+  min-width: fit-content;
+  max-width: 580px;
```

Comment:
`max-width`  is based on the width of the parent container, so I wanted to avoid a dynamic value like a percentage that could shrink it prematurely. 

`580px` was just based on feel while experimenting with different page widths to find a width that was smaller than the entire claims container, but not so big as to make the cards look oversized. We should avoid straight pixel values, though, and instead use `rem`s so it scales appropriately with browser zooms.  I'm switching it to `35rem`, which is equivalent to 560 when the default base font is 16px (16*35=560). That's just a slight adjustment down from 580, which would be `36.25rem`.